### ISSUE TITLE
Use BARTScore as loss function

### ISSUE THREAD

**Issue by hoangftran** [neulab/BARTScore#18]
Hi sir, thank you for your great work.

Can I use BARTScore to calculate the loss when training Machine Translation tasks?

Thanks.

**Comment by yyy-Apple** [comment]
You can certainly use the score calculated by BARTScore as a supervision signal. But for training models, you need to make sure that your method is differentiable.
